functions/index.js

```diff
@@ -95,6 +98,19 @@ app.get('/api/device', (req, res) => {
   return res.status(200).send(allBenchmarks[bestMatchIndex]);
 });
 
+app.get('/network-memory-considerate-model', (req, res) => {
+  const ect = req.headers.ect;
+  const deviceMemory = req.headers['device-memory'];
+  console.log('[server network-memory-considerate-model request] ECT, Device Memory => ', ect, deviceMemory);
+  
+  // TODO: As this is a demo, I think it should be easy enough to change these numbers as needed in the future. -> dotenv
+  const MEMORY_LIMIT = 4; // Threshold is 4GB RAM
+  const ECT_LIMIT = '4g';
+  // inspired by https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/client-hints/#device_hints
+  const experienceType = (ect === ECT_LIMIT && deviceMemory > MEMORY_LIMIT) ? 'heavy' : 'light';
+  res.json({experienceType});
+});
+
```

Comment:
I see that we are applying this logic in two places - functions/index.js and then again in node-ect-memory-considerate-loading/functions/index.js. I may have missed something earlier, but can you describe why these are in two different places?

Reply:
Sorry for confusion. node-ect-memory-considerate-loading/functions is any longer useless. It's just for firebase testing focusing on that project. In future, we can just use the functions/index.js. I'll remove subproject/functions for all PRs. And also after merge, I'll double check the whole project again.